packages/components/src/popover/index.js

```diff
@@ -75,7 +75,10 @@ function computeAnchorRect(
 			return getRectangleFromRange( anchorRef );
 		}
 
-		if ( anchorRef instanceof window.Element ) {
+		if (
+			anchorRef?.ownerDocument &&
+			anchorRef instanceof anchorRef.ownerDocument.defaultView.Element
```

Comment:
Maybe we can just check for `.getBoundingClientRect`?

Reply:
That would probably be more direct 🙂